tox.ini

```diff
@@ -24,6 +24,20 @@ basepython = python3
 commands =
   black --check {posargs} qiskit test tools examples setup.py
   pylint -rn qiskit test tools
+  # pylint -rn --disable='invalid-name,missing-module-docstring,redefined-outer-name' examples/python/*.py
```

Comment:
```suggestion
  pylint -rn --disable='invalid-name,missing-module-docstring,redefined-outer-name' examples/python/*.py
```

Reply:
tox doesn't do globbing so this will fail....